gateway/src/main/java/org/georchestra/gateway/accounts/events/rabbitmq/RabbitmqAccountCreatedEventSender.java

```diff
@@ -48,19 +48,22 @@ public void on(AccountCreated event) {
         if (null != oAuth2ProviderId) {
             String fullName = user.getFirstName() + " " + user.getLastName();
             String email = user.getEmail();
-            String provider = oAuth2ProviderId;
-            sendNewOAuthAccountMessage(fullName, email, provider);
+            String organization = user.getOrganization();
+            String[] providerFields = oAuth2ProviderId.split(";");
+            sendNewOAuthAccountMessage(fullName, email, organization, providerFields[0], providerFields[1]);
```

Comment:
no OOB checks ? (`ArrayIndexOutOfBoundsException`)

Reply:
You're right about the goal of this change, and yes, the semicolon is a convention in the gateway for this field. I can add OOB check, I prefer with a if on length.